builtin/credential/okta/path_config.go

```diff
@@ -310,7 +311,7 @@ func (new *oktaShimOld) Do(req *http.Request, v interface{}) (interface{}, error
 }
 
 // OktaClient creates a basic okta client connection
-func (c *ConfigEntry) OktaClient() (oktaShim, error) {
+func (c *ConfigEntry) OktaClient(ctx context.Context) (oktaShim, context.Context, error) {
```

Comment:
Looks like the call to `OktaClient` doesn't use the returned context, and the context returned here could be accessed by `Client()`. I think we can simply not return that context here.

Reply:
Fixed